providers/base/fetch.go

```diff
@@ -80,7 +76,8 @@ func (p *Provider[K, V]) startAPI(ctx context.Context, responseCh chan<- provide
 // attemptAPIDataUpdate tries to update data by fetching and parsing API data.
 // It logs any errors encountered during the process.
 func (p *Provider[K, V]) attemptAPIDataUpdate(ctx context.Context, responseCh chan<- providertypes.GetResponse[K, V]) {
-	if len(p.ids) == 0 {
+	ids := p.GetIDs()
```

Comment:
Unclear what p.GetIDs() is?

Reply:
IDs are effectively currency pairs, the base provider just calls them IDs to have some generality.